### PR TITLE
deno: build for Apple Silicon

### PR DESCRIPTION
**Pull request by fxcoudert**
One of the most-downloaded formula with no Apple Silicon bottle… let's try to fix it

### CODE REVIEW

File: Formula/deno.rb [hunk 1]
```diff
@@ -4,6 +4,7 @@ class Deno < Formula
   url "https://github.com/denoland/deno/releases/download/v1.6.0/deno_src.tar.gz"
   sha256 "60491d842e04ce162face61bb8857bf18a41726afbcbcd9fa532055ace7431ae"
   license "MIT"
+  revision 1
```
Comment (SMillerDev) [inline Formula/deno.rb:7]: bumping the version might be better.

**Comment by SMillerDev** [comment]
Following the chain of linked threads I don't think python 2 (and therefore macOS) is a hard requirement anymore.

**Comment by carlocab** [comment]
Python2 is a hard requirement for building V8 from source

**Comment by SMillerDev** [comment]
> Python2 is a hard requirement for building V8 from source

Not according to the formula: https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/v8.rb

**Comment by fxcoudert** [comment]
```
ld: in /private/tmp/deno-20210107-61596-bjk89x/deno/target/release/deps/librusty_v8-a13c2e74ac9dc1eb.rlib(binding.o),
archive member 'binding.o' with length 160944 is not mach-o or llvm bitcode for architecture arm64
```

**Comment by carlocab** [comment]
The formula doesn't matter, as this is what deno uses:

https://github.com/denoland/rusty_v8#build-v8-from-source

**Comment by bayandin** [comment]
> > Python2 is a hard requirement for building V8 from source
> 
> Not according to the formula: https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/v8.rb

Actually, we use python2 to build `v8`:
https://github.com/Homebrew/homebrew-core/blob/c2bd025d4ddd1f25434055eff63272eb11163296/Formula/v8.rb#L82

**Comment by asip** [comment]
Is this issue not useful ?
https://github.com/denoland/deno/issues/8992

**Comment by carlocab** [comment]
Bottled in #66920.

**Pull request closed without merging**